### PR TITLE
update deprecated tree walk endpoint

### PR DESCRIPTION
**Pull request by kemartin2015**

### CODE REVIEW

File: archivessnake/notes_csv.py [hunk 1]
```diff
@@ -7,40 +7,52 @@
 import time
 
 from asnake.aspace import ASpace
+from asnake.utils import walk_tree
 
 config = ConfigParser()
 config.read("local_settings.cfg")
 
 aspace = ASpace(baseurl=config.get('ArchivesSpace', 'baseURL'), username=config.get('ArchivesSpace', 'user'), password=config.get('ArchivesSpace', 'password'))
+client = aspace.client
```
Comment (helrond) [inline archivessnake/notes_csv.py]: You could streamline this by just doing:
```suggestion
client = ASpace(baseurl=config.get('ArchivesSpace', 'baseURL'), username=config.get('ArchivesSpace', 'user'), password=config.get('ArchivesSpace', 'password')).client
```

File: archivessnake/notes_csv.py [hunk 2]
```diff
@@ -7,40 +7,52 @@
...
+                                container = "None"
+                            try:
+                                container_2 = "{} {}".format(sub_container['type_2'].capitalize(), sub_container['indicator_2'])
+                            except KeyError:
+                                container_2 = "None"
+                            location = ""
+                            for loc in top_container['container_locations']:
                                 try:
-                                    container_2 = "{} {}".format(sub_container.type_2.capitalize(), sub_container.indicator_2)
+                                    l = aspace.client.get(loc['ref']).json()
```
Comment (helrond) [inline archivessnake/notes_csv.py]: you can just reference the `client` variable here.

File: archivessnake/notes_csv.py [hunk 3]
```diff
@@ -7,40 +7,52 @@
...
+                            expression = record['dates'][0]['expression']
                         except (KeyError, IndexError):
                             expression = "undated"
-                        for instance in record.instances:
-                            sub_container = instance.sub_container
-                            top_container = instance.sub_container.top_container
-                            for loc in top_container.container_locations:
+                        for instance in record['instances']:
+                            sub_container = instance['sub_container']
+                            top_container = aspace.client.get(instance['sub_container']['top_container']['ref']).json()
```
Comment (helrond) [inline archivessnake/notes_csv.py]: You can reference the `client` variable here, rather than `aspace.client`

**Review by helrond — CHANGES_REQUESTED** [review]
Looks good - there are some opportunities to streamline the variables (looks like you don't need _both_ a `client` and `aspace` variable since you are referencing `aspace.client`, which is the same thing as `client`).

**Review by p-galligan — APPROVED** [review]
Looks good to me aside from the comments already mentioned.

**Comment by kemartin2015** [comment]
Thank you both! I removed the references to aspace.

**Review by helrond — APPROVED** [review]

**Pull request merged**